.github/workflows/test-build.yml

```diff
@@ -57,21 +19,13 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
-      - name: Cache Poetry cache
-        uses: actions/cache@v2
-        with:
-          path: ~/.cache/pypoetry
-          key: poetry-cache-${{ runner.os }}-${{ matrix.python-version }}-${{ env.POETRY_VERSION }}
-
       - name: Set up Python
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
 
       - name: Install Poetry
-        run: |
-          curl -sSL ${{ env.POETRY_URL }} | python -
-          echo "$HOME/.local/bin" >> $GITHUB_PATH
+        uses: ThePalaceProject/circulation/.github/actions/poetry@main
```

Comment:
Not urgent, but should we eventually move this to `ci-scripts` since it is shared? And maybe rename `ci-scripts` to `ci-shared`?